client/src/pages/users/Users.jsx

```diff
@@ -42,11 +42,11 @@ function Users () {
   return (
     <Container>
       <InviteModal opened={opened} close={close} />
-      <Group justify='space-between' wrap='nowrap' my='sm'>
-        <Title order={3} mr='md'>
+      <Group justify='space-between' my='sm' className={classes.searchContainer}>
```

Comment:
Could utilize Mantine's default behavior for the `Group` component, where if there's no prop for `wrap`, it defaults to the equivalent `flex-wrap: wrap`